core-bundle/src/Resources/contao/dca/tl_form.php

```diff
@@ -130,6 +130,8 @@
 	(
 		'id' => array
 		(
+			'label'                   => array('ID'),
+			'search'                  => true,
```

Comment:
If we are going to add this everywhere, we should really add it everywhere. People might want to search news, events, users, members, FAQs etc. by ID, too.

Reply:
Then we can just add this in the `DC_Table` by default.